src/revlanguage/distributions/phylogenetics/branchrate/Dist_PhyloOrnsteinUhlenbeckPruning.cpp

```diff
@@ -132,17 +132,17 @@ const TypeSpec& Dist_PhyloOrnsteinUhlenbeckREML::getClassTypeSpec(void)
  *
  * \return Rev name of constructor function.
  */
-std::string Dist_PhyloOrnsteinUhlenbeckREML::getDistributionFunctionName( void ) const
+std::string Dist_PhyloOrnsteinUhlenbeckPruning::getDistributionFunctionName( void ) const
 {
     // create a distribution name variable that is the same for all instance of this class
-    std::string d_name = "PhyloOrnsteinUhlenbeckREML";
+    std::string d_name = "PhyloOrnsteinUhlenbeckPruning";
```

Comment:
This is a bit dangerous to do as it breaks backwards compatibility. Would it be OK to keep `PhyloOrnsteinUhlenbeckREML` as an alias? This is imply from the `Rev` side of things.

Reply:
That sounds good to me!